packages/language-server/server/utils/document_ast.js

```diff
@@ -0,0 +1,28 @@
+const babylon = require("@babel/parser");
```

Comment:
And could I ask you to give a name to documents in kebab case: `document-ast.js`?